### PR TITLE
nftest for freec

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/controlfreec/freec/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    process.out.bedgraph,
+                    process.out.control_cpn,
+                    process.out.sample_cpn,
+                    process.out.gcprofile_cpn,
+                    process.out.BAF,
+                    process.out.CNV,
+                    process.out.info,
+                    process.out.ratio,
+                    process.out.versions).match() },
+                 { assert process.out.config }
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply modules/nf-core/controlfreec/freec/tests/main.nf.test:64]: that it exists according to @sateeshperi

[the remark this answers is not in the document]
Comment (sateeshperi) [reply modules/nf-core/controlfreec/freec/tests/main.nf.test:64]: yup....any issue with that @maxulysse ?

File: modules/nf-core/controlfreec/freec/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                input[8] = file(params.test_data['homo_sapiens']['genome']['genome_21_multi_interval_bed'], checkIfExists: true)
+                input[9] = []
+                """
+            }
+        }
+
+        then {
+            assertAll (
+                { assert process.success }
+            )
```
Comment (sateeshperi) [inline modules/nf-core/controlfreec/freec/tests/main.nf.test:111]: need snap assertions here

Comment (FriederikeHanssen) [reply modules/nf-core/controlfreec/freec/tests/main.nf.test:111]: completely forgot about this PR. I am running stub tests here so all the files will be empty. _I think_ this is why I thought I don't need the assertion. What is the best practice here?

Comment (SPPearce) [reply modules/nf-core/controlfreec/freec/tests/main.nf.test:111]: The tools template is `{ assert snapshot(process.out).match() }`

**Comment by SPPearce** [comment]
Superceded by #5398.

**Pull request closed without merging**